charts/astronomer/values.yaml

```
@@ -171,6 +171,24 @@ houston:
     # Only run on deployments marked as canary
     canary: false
 
+  # Cleanup airflow db data
+  # This runs as a CronJob
+  cleanupAirflowDb:
+    # Enable cleanup CronJob
+    enabled: true
+
+    # Default here is to run at midnight every night https://crontab.guru/#0_0_*_*_*
+    schedule: "0 0 * * *"
```

Let's use anything except 0 in order to avoid running at the same time as so many other jobs that are scheduled on the 0 minute and hour.

```suggestion
    # https://crontab.guru/#23_5_*_*_*
    schedule: "23 5 * * *"
```